client/components/domains/domain-product-price/index.jsx

```diff
@@ -137,6 +137,26 @@ class DomainProductPrice extends Component {
 		);
 	}
 
+	// This method returns "Free domain for one year" text (different from "Free with plan")
+	renderFreeForFirstYear() {
+		const { translate } = this.props;
+
+		const className = classnames( 'domain-product-price', 'is-free-domain', {
+			'domain-product-price__domain-step-signup-flow': this.props.showStrikedOutPrice,
+		} );
+
+		return (
+			<div className={ className }>
+				<div className="domain-product-price__free-text">
+					<span className="domain-product-price__free-price">
+						{ translate( 'Free domain for one year' ) }
```

Comment:
What do you think about using `Free for first year`? That was the text used on Gravatar's side as well, if I remember correctly.

It also matches the method name :)